qa/zcash/updatecheck.py

```diff
@@ -89,32 +95,45 @@ def get_dependency_list():
             DependsVersionGetter("utfcpp"))
     ]
 
-    # Rust crates.
+    # Rust crates (filename portion: depends/packages/crate_<NAME>.mk).
     crates = [
-        "aes", "aesni", "aes_soft", "arrayvec", "bellman",
-        "arrayref", "autocfg", "bigint", "blake2b_simd", "blake2s_simd",
-        "bit_vec", "block_cipher_trait", "byteorder",
-        "block_buffer", "block_padding", "c2_chacha", "cfg_if", "crunchy",
-        "byte_tools", "constant_time_eq", "crossbeam", "digest", "fpe",
+        "aes", "aesni", "aes_soft", "arrayvec", "bellman", "arrayref",
+        "autocfg", "bigint", "blake2b_simd", "blake2s_simd", "bit_vec",
+        "block_cipher_trait", "byteorder", "byte_tools", "block_buffer",
+        "block_padding", "c2_chacha", "cfg_if", "crunchy",
```

Comment:
Please change this list to one entry per line, to make future changes easier to review.